### DISCUSSION TITLE
The M chip's macOS cannot install rdkit-stubs through conda-forge.

### DISCUSSION THREAD

**Discussion by mrzzmrzz** [rdkit/rdkit#7404] [Q&A]
My MacBook is equipped with the M1 Max chip, and the system version is macOS 14.4.1. I found that the rdkit installed through conda does not have rdkit-stubs, but the rdkit installed through pip has rdkit-stubs.

I used another MacBook with an Intel chip, which stayed on macOS 12.7.4, and the conda-installed rdkit on this version has rdkit-stubs. This is very strange. Can someone please help me solve this problem?

**Comment by ptosco** [comment]
Thanks for reporting. It looks like none of the current macOS arm64 conda packages contains the `rdkit-stubs` directory.
I'll check what's wrong and fix it for next RDKit patch release.

**Reply by mrzzmrzz** [reply]
Thank you very much! I'm really looking forward to the subsequent updates and fixes.
